airflow-core/src/airflow/ui/public/i18n/locales/de/admin.json

```
@@ -36,6 +36,7 @@
       "extraFields": "Weitere Felder",
       "extraFieldsJson": "JSON Ansicht der weiteren Felder",
       "helperText": "Fehlt ein Verbindungstyp? Stellen Sie sicher, dass die notwendigen Provider-Pakete installiert sind.",
+      "helperTextForRedactedFields": "Redigierte Felder ('***') bleiben unverändert, wenn sie nicht bearbeitet werden.",
```

```suggestion
      "helperTextForRedactedFields": "Unkenntlich gemachte Felder ('***') bleiben unverändert, wenn sie nicht bearbeitet werden.",
```

Ich musste 'redigiert' erstmal nachschauen 😅 ich glaube so ist es präziser und etwas einfacher zu verstehen was gemeint ist.